ssp.md

```diff
@@ -3,18 +3,18 @@ layout: index
 title: GCAM Shared-Socioeconomic Pathways
 prev: macro-econ.html
 next: energy.html
-gcam-version: v5.0 
+gcam-version: v5.1 
 ---
 
 ## Overview of the SSPs
 
-The GCAM model was one of six models used in the quantification of the Shared Socioeconomic Pathways (SSP; Riahi et al., 2017). The SSPs are new reference scenarios for climate change modeling and research (Moss et al. 2010, van Vuuren et al., 2014). That is the five SSPs assume no additional policies and measures explicitly designed to reduce greenhouse gas emissions.  However, each of the five reference scenarios can be paired with a set of emissions mitigation assumptions.  Thus, when paired with a limit of 2.6, 4.5, or 6.0, a constrained SSP can be paired with the corresponding climate model experiments in the CMIP6 data base. The five SSPs are defined along two different axes: challenges to mitigation and challenges to adaptation (see Figure below).
+The GCAM model was one of six models used in the quantification of the Shared Socioeconomic Pathways (SSP; Riahi et al., 2017). The SSPs are new reference scenarios for climate change modeling and research (Moss et al. 2010, van Vuuren et al., 2014). That is the five SSPs assume no additional policies and measures explicitly designed to reduce greenhouse gas emissions.  However, each of the five reference scenarios can be paired with a set of emissions mitigation assumptions.  Thus, when paired with a limit of 2.6, 4.5, or 6.0, the resulting scenario can be used in combination with the corresponding climate model experiments in the CMIP data base. The five SSPs are defined along two different axes: challenges to mitigation and challenges to adaptation (see Figure below).
```

Comment:
This whole paragraph is actually a single line, which makes it very difficult to tell what the actual change was.  Theoretically github should highlight the actual words that were changed (like it does in the next paragraph below), but apparently the highlighter gets confused if the change is too far into a really, really long line.  

These markdown files would be a lot easier to work with if we wrapped them to some reasonable length (80 col, 120 col, whatever).  These unwrapped lines are all throughout the documentation, so it's probably too late to go back and change them all, but it's something to keep in mind going forward.

Reply:
@rplzzz: I wrap them in BBedit so that I can read them. How do I make it so the wrapping is committed (other than manually)?

Reply:
I'm not sure; I've never used BBedit.  My guess is that it's wrapping them visually, but not actually modifying the text.  Maybe look for a command with a name like "fill paragraph" or "reflow paragraph"?  If it has a mode to autofill when a line reaches a certain length, then that's even better, since that would fill as you type without you having to do it manually.